cpp/velox/compute/VeloxParquetDatasource.h

```diff
@@ -34,6 +34,8 @@
 #include "operators/writer/Datasource.h"
 
 #include "velox/common/file/FileSystems.h"
+#include "velox/connectors/hive/storage_adapters/hdfs/HdfsFileSink.h"
```

Comment:
I think we should use macro `#ifdef ENABLE_HDFS`. like  https://github.com/oap-project/gluten/blob/main/cpp/velox/compute/VeloxInitializer.cc#L28-L30